### DISCUSSION TITLE
Add 'Saved' and 'Multiple Collections' Options

### DISCUSSION THREAD

**Discussion by HasiburRahmaan** [dailydotdev/daily#1650] [Feature Requests]
I’d like to suggest a new feature that could enhance user experience: a 'Saved' section with support for organizing content into multiple collections (similar to Facebook’s approach).
I often come across articles I want to save for later. Having the ability to organize these saved articles into multiple collections would help me keep them categorized for quick access, e.g., 'Tech Trends,' 'Programming Tips,' or 'Career Advice.'

**Comment by idoshamun** [comment]
Hi @HasiburRahmaan,

Please check this thread https://github.com/dailydotdev/daily/discussions/520
